docs/guides/mysql/clustering/group_replication_single_primary.md

```diff
@@ -0,0 +1,489 @@
+  IP:           10.107.10.177
+  Port:         db  3306/TCP
+  TargetPort:   db/TCP
+  Endpoints:    172.17.0.5:3306,172.17.0.6:3306,172.17.0.7:3306
+
+Service:
+  Name:         my-group-gvr
+  Labels:         app.kubernetes.io/component=database
+                  app.kubernetes.io/instance=my-group
+                  app.kubernetes.io/managed-by=kubedb.com
+                  app.kubernetes.io/name=mysql
+                  app.kubernetes.io/version=5.7.25
+                  kubedb.com/kind=MySQL
+                  kubedb.com/name=my-group
+  Annotations:    service.alpha.kubernetes.io/tolerate-unready-endpoints=true
+  Type:         ClusterIP
+  IP:           None
+  Port:         db  3306/TCP
+  TargetPort:   3306/TCP
+  Endpoints:    172.17.0.5:3306,172.17.0.6:3306,172.17.0.7:3306
+
+Database Secret:
+  Name:         my-group-auth
+  Labels:         kubedb.com/kind=MySQL
+                  kubedb.com/name=my-group
+  Annotations:  <none>
+
+Type:  Opaque
+
+Data
+====
+  password:  16 bytes
+  username:  4 bytes
+
+No Snapshots.
+
+Events:
+  Type    Reason      Age   From            Message
+  ----    ------      ----  ----            -------
+  Normal  Successful  47m   MySQL operator  Successfully created Service
+  Normal  Successful  43m   MySQL operator  Successfully created StatefulSet
+  Normal  Successful  43m   MySQL operator  Successfully created MySQL
+  Normal  Successful  43m   MySQL operator  Successfully created appbinding
+  Normal  Successful  43m   MySQL operator  Successfully patched StatefulSet
+  Normal  Successful  43m   MySQL operator  Successfully patched MySQL
+
+$ kubectl get statefulset -n demo
+NAME       READY   AGE
+my-group   3/3     49m
+
+$ kubectl get pvc -n demo
+NAME              STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+data-my-group-0   Bound    pvc-ea20656d-6809-11e9-89c6-080027fc7fb2   1Gi        RWO            standard       49m
+data-my-group-1   Bound    pvc-4a2d43b0-680a-11e9-89c6-080027fc7fb2   1Gi        RWO            standard       47m
+data-my-group-2   Bound    pvc-60558ef0-680a-11e9-89c6-080027fc7fb2   1Gi        RWO            standard       46m
+
+$ kubectl get pv -n demo
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                  STORAGECLASS   REASON   AGE
+pvc-4a2d43b0-680a-11e9-89c6-080027fc7fb2   1Gi        RWO            Delete           Bound    demo/data-my-group-1   standard                56m
+pvc-60558ef0-680a-11e9-89c6-080027fc7fb2   1Gi        RWO            Delete           Bound    demo/data-my-group-2   standard                55m
+pvc-ea20656d-6809-11e9-89c6-080027fc7fb2   1Gi        RWO            Delete           Bound    demo/data-my-group-0   standard                59m
+
+$ kubectl get service -n demo
+NAME           TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+my-group       ClusterIP   10.107.10.177   <none>        3306/TCP   59m
```

Comment:
- Does this service only point to master pod?
- How do users connect to the read replicas?